src/components/navigation/Navbar.stories.tsx

```diff
@@ -161,6 +176,54 @@ export const CustomChildElement: Story = {
   },
 };
 
+export const LinksInSlot: Story = {
+  args: {
+    rightSlot: [
+      <User
+        key="user"
+        onLogin={() => {}}
+        onLogout={() => {}}
+        user={{ name: "Name", fedid: "FedID" }}
+        color={"white"}
+      />,
+      <ColourSchemeButton key="colourScheme" />,
+    ],
+    logoRightSlot: [
+      <NavLinks key="links">
+        <NavLink href="#Mercury" key="mercury">
+          Mercury
+        </NavLink>
+        <NavLink href="#Venus" key="venus">
+          Venus
+        </NavLink>
+        <NavLink href="#Earth" key="earth">
+          Earth
+        </NavLink>
+        <NavLink href="#Mars" key="mars">
+          Mars
+        </NavLink>
+      </NavLinks>,
+    ],
+    logo: "theme",
+  },
+};
+
+export const AllSlots: Story = {
```

Comment:
In Storybook and clicking on "Show Code" this wierdly generates:

```
<Navbar
  centreSlot={<React.ForwardRef sx={{color: 'white'}}>centre</React.ForwardRef>}
  logo="theme"
  logoLeftSlot={<React.ForwardRef sx={{color: 'white'}}>logo left</React.ForwardRef>}
  logoRightSlot={<React.ForwardRef sx={{color: 'white'}}>logo right</React.ForwardRef>}
  rightSlot={<React.ForwardRef sx={{color: 'white'}}>right</React.ForwardRef>}
>
  <React.ForwardRef
    sx={{
      color: 'white'
    }}
  >
    children
  </React.ForwardRef>
</Navbar>
```

Any ideas why it is using `React.ForwardRef`?

Reply:
I'm not seeing that. Do you still see it with the latest version?